src/attr/_funcs.py

```diff
@@ -22,10 +22,15 @@ def asdict(inst, recurse=True, filter=None):
         value as the second argument.
     :type filer: callable
 
+    :param dict_factory: A callable to produce dictionaries from. For example,
+        to produce ordered dictionaries instead of normal Python dictionaries,
+        pass in ``collections.OrderedDict``.
+    :type dict_factory: callable
```

Comment:
this should be written as `:param callable dict_factory:`.  Bonus points if you fix it above too. :)